documentation/book/ref-metrics-alertmanager-examples.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-prometheus-alertmanager.adoc
+[id='ref-metrics-alertmanager-examples-{context}']
+
+= Alerting rule examples
+
+Example alerting rules for Kafka and Zookeeper metrics are provided with {ProductName} for use in a xref:proc-metrics-deploying-prometheus-{context}[Prometheus deployment].
+
+The rules are defined in a `PrometheusRule` resource in the link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`^] file
+
+.Kafka alerting rules
+
+* `UnderReplicatedPartitions`: gives the number of partitions for which the current broker is the lead replica but the follower replicas are behind.
+This metric provides insights about offline brokers that host the follower replicas.
+An alert is raised when this value is greater than zero, providing information on the under-replicated partitions for each broker.
+
+* `AbnormalControllerState`: indicates whether the current broker is the controller for the cluster.
+The metric can be 0 or 1.
+During the life of a cluster, only one broker should be the controller and the cluster always needs to have an active controller.
+Having two or more brokers saying that they are controllers indicates a problem.
+An alert is raised when the sum of all the values for this metric on all brokers is not equal to 1, meaning that there is no active controller (the sum is 0) or more than one controller (the sum is greater than 1).
```

Comment:
@ppatierno is it raised immediately? What happens during controller election?

Reply:
The alerts should be raised if the condition persists for some time which is configurable IIRC. Right @stanlyDoge ?

Reply:
Yes, when the condition is fulfilled there is some time bore the alert is fired. It is set by option 
`for: 10s`

Reply:
Updated to mention the `for` property